icechunk/src/virtual_chunks.rs

```diff
@@ -58,6 +62,9 @@ impl VirtualChunkContainer {
             (ObjectStoreConfig::LocalFileSystem(..), _) => {
                 Err("in memory storage does not accept credentials".to_string())
             }
+            (ObjectStoreConfig::Http(_), _) => {
+                Err("http storage does not support credentials".to_string())
```

Comment:
Should we support basic auth at least? Is there a reason we cannot or simply not implementing in this PR? If so, please add a TODO in the code and a ticket